opencti-platform/opencti-front/src/private/components/settings/support/SupportPackageLine.tsx

```diff
@@ -0,0 +1,280 @@
+import { hexToRGB } from '../../../../utils/Colors';
+import { DataColumns } from '../../../../components/list_lines';
+
+const useStyles = makeStyles(() => ({
+  bodyItem: {
+    height: 20,
+    fontSize: 13,
+    float: 'left',
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    paddingRight: 10,
+  },
+  item: {
+    paddingLeft: 10,
+    height: 50,
+    cursor: 'default',
+  },
+  chipInList: {
+    fontSize: 12,
+    height: 20,
+    float: 'left',
+    borderRadius: 4,
+    width: 120,
+  },
+  label: {
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+  },
+}));
+
+type PackageStatus = 'IN_PROGRESS' | 'READY' | 'IN_ERROR' | '%future added value';
+
+const SupportPackageLineForceZipMutation = graphql`
+  mutation SupportPackageLineForceZipMutation(    
+    $input: SupportPackageForceZipInput!
+  ) {
+      supportPackageForceZip(input: $input) {
+        id
+      }
+  }
+`;
+
+const SupportPackageLineDeleteMutation = graphql`
+  mutation SupportPackageLineDeleteMutation($id: ID!) {
+    supportPackageDelete(id: $id)
+  }
+`;
+
+export const supportPackageLineFragment = graphql`
+  fragment SupportPackageLine_node on SupportPackage {
+    id
+    name
+    package_status
+    package_url
+    package_upload_dir
+    created_at
+    creators {
+      id
+      name
+    }
+  }
+`;
+
+const packageStatusColors: { [key in PackageStatus]: string } = {
+  IN_PROGRESS: '#303f9f',
+  READY: '#4caf50',
+  IN_ERROR: '#f44336',
+  '%future added value': '#9e9e9e',
+};
+
+interface SupportPackageLineProps {
+  dataColumns: DataColumns;
+  node: SupportPackageLine_node$key;
+  paginationOptions: { search: string; orderMode: string; orderBy: string };
+}
+
+const SupportPackageLine: FunctionComponent<SupportPackageLineProps> = ({
+  node,
+  paginationOptions,
+  dataColumns,
+}) => {
+  const classes = useStyles();
+  const { t_i18n, fd } = useFormatter();
+  const data = useFragment(supportPackageLineFragment, node);
+  const [displayDelete, setDisplayDelete] = useState(false);
+  const [deleting, setDeleting] = useState(false);
+  const [commitDelete] = useMutation(SupportPackageLineDeleteMutation);
+  const [commitForceZip] = useMutation(SupportPackageLineForceZipMutation);
+  const isProgress = data.package_status === 'IN_PROGRESS';
+
+  const handleOpenDelete = () => {
+    setDisplayDelete(true);
+  };
+
+  const handleCloseDelete = () => {
+    setDisplayDelete(false);
+  };
+
+  const submitDelete = () => {
+    setDeleting(true);
+    commitDelete({
+      variables: {
+        id: data.id,
+      },
+      updater: (store: RecordSourceSelectorProxy) => deleteNode(
+        store,
+        'Pagination_supportPackages',
+        paginationOptions,
+        data.id,
+      ),
+      onCompleted: () => {
+        setDeleting(false);
+        handleCloseDelete();
+      },
+      onError: (error: Error) => {
+        handleError(error);
```

Comment:
should we not set deleting to false also in case of error? In any case the request is done